website/content/en/docs/scorecard/scorecard-alpha.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Scorecard (alpha)
+weight: 25
+---
+
+
+## Overview
+
+The scorecard command, part of the operator-sdk, executes tests 
+on your operator based upon a configuration file and test images.
+
+Tests are implemented within test images that are configured 
+and constructed to be executed by scorecard.
+
+Scorecard assumes it is being executed with access to a configured
+Kube cluster.  Each test is executed within a Pod by scorecard, 
+from which pod logs are aggregated and test results sent to the console. 
+
+Scorecard has built-in basic and OLM tests but also provides a 
+means to execute custom test definitions.
+
+## Requirements
+
+The scorecard tests make no assumptions as to the state of the 
+operator being tested.  Creating operators and custom resources 
+for an operator are left outside the scope of the scorecard itself.
+
+Scorecard tests can however create whatever resources they 
+require if the tests are designed for resource creation.
+
+## Running the Scorecard
+
+1. Define a scorecard configuration file `config.yaml`.  A sample 
+configuration file [config.yaml][sample-config] is found within 
+the SDK repository. See the [config file section](#config-file) for an explaination 
+of the configuration file format.  Unless you are executing custom 
+tests, you can just copy the provided example configuration file 
+into your project.
+2. Place the scorecard configuration file within your project 
+bundle directory a the following location `tests/scorecard/config.yaml`.  
+You can override the default location of the configuration file 
+by specifying the `--config` flag.
+3. Execute the [`scorecard` command][cli-scorecard]. See the 
+[command args section](#command-args) for an overview of command invocation.
+
+## Configuration
+
+The scorecard test execution is driven by a configuration file typically named
+`config.yaml`. The configuration file is located at the following 
+location within your bundle:
+```
+tests/scorecard/config.yaml
+```
+
+### Config File
+
+A complete scorecard configuration file can be found [here][sample-config] 
+and used for running the scorecard pre-defined tests that ship with the SDK.
+
+A sample of the scorecard configuration file may look as follows:
+
+```yaml
+tests:
+- name: "basic-check-spec"
+  image: quay.io/operator-framework/scorecard-test:dev
```

Comment:
Hi @jmccormick2001 just realized. 
- This image does not exist
- And then, should we not have tags which match with the SDK ones instead of it?